LogKitten.java

```diff
@@ -46,7 +47,7 @@
 			// Create new file if it doesn't exist (this should happen)
 			file.createNewFile(); // creates if does not exist
 			// Create FileOutputStream to actually write to the file.
```

Comment:
You might consider changing this comment but I think it's ok because the BufferedOutputStream wraps a FileOutputStream.